openapi.yaml

```diff
@@ -171,6 +547,365 @@ components:
           schema:
             $ref: '#/components/schemas/ErrorResult'
   schemas:
+    BookId:
+      type: string
+      description: Book ID
+      example: '26'
+    BookType:
+      type: string
+      enum:
+        - book
+    BookAvailable:
+      type: string
+      nullable: false
+      description: Availability of book (true/false).
+      example: true
+    MemberId:
+      type: string
+      description: Member ID
+      example: '001'
+    MemberType:
+      type: string
+      enum:
+        - member
+    MembershipStatus:
+      type: string
+      nullable: false
+      description: Membership status of the member (active/suspended/expired).
+      example: active
+    BorrowId:
+      type: string
+      description: Borrow ID
+      example: 'bo123'
+    BorrowType:
+      type: string
+      enum:
+        - borrow
+    BorrowReturnDate:
+      type: string
+      format: date
+      nullable: true
+      description: Returned date of the book
+      example: null
```

Comment:
The main purpose of `example` field is to allow your user to know what to expect by reading this API spec file, so we usually won't put `null` as an example (even though it's a nullable field) since it only helps a little.

Reply:
done